src/frontend/komponenter/ManuellJournalfør/KnyttJournalpostTilBehandling.tsx

```diff
@@ -65,68 +68,57 @@ export const KnyttJournalpostTilBehandling: React.FC = () => {
-                                                    ]
-                                                }
-                                            </Table.DataCell>
-                                            <Table.DataCell>
-                                                {behandlingstyper[behandling.type].navn}
-                                            </Table.DataCell>
-                                            <Table.DataCell>
-                                                {behandlingsstatuser[behandling.status]}
-                                            </Table.DataCell>
-                                        </Table.Row>
-                                    )
+                                    <Table.Row
+                                        key={behandling.id}
+                                        aria-selected={skjema.felter.tilknyttedeBehandlingIder.verdi.includes(
+                                            behandling.id
+                                        )}
+                                    >
+                                        <Table.DataCell>
+                                            <Checkbox
+                                                id={behandling.id}
+                                                value={behandling.id}
+                                                checked={skjema.felter.tilknyttedeBehandlingIder.verdi.includes(
+                                                    behandling.id
+                                                )}
+                                                onChange={() => {
+                                                    skjema.felter.tilknyttedeBehandlingIder.validerOgSettFelt(
+                                                        [
+                                                            ...skjema.felter.tilknyttedeBehandlingIder.verdi.filter(
+                                                                it => it !== behandling.id
+                                                            ),
+                                                            ...(skjema.felter.tilknyttedeBehandlingIder.verdi.includes(
+                                                                behandling.id
+                                                            )
+                                                                ? []
+                                                                : [behandling.id]),
+                                                        ]
+                                                    );
+                                                }}
+                                                readOnly={!kanKnytteJournalpostTilBehandling()}
+                                                hideLabel={true}
+                                            >
+                                                {behandling.id}
```

Comment:
Den nye koden her er mye enklere og finere å lese nå som `behandling.id` er en string i visningen. I stedet for å ha `behandling.behandlingId.toString()` over alt i koden 🎉